### PR TITLE
temperature@fevimu: Adding label prefix support to applet panel display

### PR DESCRIPTION
**Pull request by dylanparker**
Simple change to add a customizable prefix to the temperate display in the panel.
Default included prefix (disabled by default) is the Unicode thermometer character (U+1F321). 

screencap

Triggering a notification to @claudiux

### CODE REVIEW

File: temperature@fevimu/files/temperature@fevimu/settings-schema.json [hunk 1]
```diff
@@ -22,6 +22,20 @@
...
+    "tooltip": "Whether or not to add a label before the temperature value."
+  },
+  "label-prefix": {
+    "type": "entry",
+    "default": "🌡 ",
+    "description": "Label prefix",
+    "indent": true,
+    "tooltip": "The label to be shown before the temperature value.",
+    "dependency": "show-label-prefix"
+  },
```
Comment (claudiux) [inline temperature@fevimu/files/temperature@fevimu/settings-schema.json:38]: Please remove the "indent" line (line 35) and the blank space of the "default" field.

File: temperature@fevimu/files/temperature@fevimu/applet.js [hunk 2]
```diff
@@ -255,6 +257,10 @@ CPUTemperatureApplet.prototype = {
       }
     }
 
+    if (this.state.showLabelPrefix) {
+      this.title = this.state.labelPrefix + this.title;
```
Comment (claudiux) [inline temperature@fevimu/files/temperature@fevimu/applet.js:261]: Please replace lien 261 by:
```
      this.title = "%s %s".format(this.state.labelPrefix, this.title);
```

**Comment by claudiux** [comment]
@dylanparker Thank you. Good idea! Can you change your code as I requested, please ?

**Comment by brownsr** [comment]
Hi Dylan, waiting for you to respond to claudiux ...

**Comment by dylanparker** [comment]
Hey @claudiux , updated as requested. Thanks for the review.
I would prefer to have the space after the prefix optional but understand my original approach was clumsy with the extra space in the 'default' field. Would you be against just having the string format on line 261 be represented by "%s%s" without the space?

**Pull request merged**